package.json

```diff
@@ -18,6 +18,7 @@
     "prettier:diff": "prettier --list-different \"**/*.{scss,css,js,md}\"",
     "semantic-release": "semantic-release",
     "start": "yarn storybook",
+    "start:experimental": "cross-env CARBON_USE_EXPERIMENTAL_FEATURES=true yarn start",
```

Comment:
Instead of adding as a script can we just add it to a md file under `docs`? This will be more of a temporary thing so would rather it live there instead of code.

Reply:
Hmm, could we leave it here for now and remove it later? This makes it a lot easier while developing. Yes its "temporary" but still will be around for ~6 months or so most likely.

Reply:
yeah I initially added it for ease of development and would like to keep it for the time being, but I can also create a local gitignored script to start the experimental environment if that would be better

Reply:
@joshblack @alisonjoseph I've removed this change from this PR and created a separate PR for it here https://github.com/IBM/carbon-components-react/pull/1547